apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -8,47 +8,35 @@ import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
 
 export const COLUMNS = {
   QUESTION: 0,
-  ANSWER_1: 1,
-  ANSWER_2: 2,
-  NOT_ANSWERED: 3,
 };
 
-const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionOne}
-        isCorrectAnswer={rowData.optionOneIsCorrect}
-      />
-  );
-};
+const alphabetMapper =  [
```

Comment:
indentation should look more like the following to match style
```
+const alphabetMapper =  [
  commonMsg.answerOptionA(),
  commonMsg.answerOptionB(),
  commonMsg.answerOptionC(),
  commonMsg.answerOptionD(),
  commonMsg.answerOptionE(),
  commonMsg.answerOptionF(),
  commonMsg.answerOptionG(),
];
```

Reply:
Fixed.